security.md

```
@@ -3,13 +3,13 @@
 ## Supported Versions
 
 While Pelican is in beta, we only provide security fixes for the most recent beta release. Older beta releases are unsupported.  
-![](https://img.shields.io/github/v/release/pelican-dev/panel?label=latest-release)
+![](https://img.shields.io/github/v/release/pelican/panel?label=latest-release)
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)